docker-compose.yaml

```diff
@@ -0,0 +1,29 @@
+version: '3.1'
+services:
+  app:
+    image: craftcms
+    build:
+      context: .
+      dockerfile: Dockerfile
+    ports:
+      - 8000:80
+    volumes:
+      - .:/var/www/html:cached
+    env_file: .env
```

Comment:
I've had issues with using Craft's `.env.example` due to quotes wrapping the value, which docker-compose will include as part of the env var value. Not sure what the appropriate actions here are, but I have started manually removing quotes from the `.env` when using the `env_file` config.

Reply:
That is a good point, I have seen it with boolean and special chars a lot... I think that this request would need a section on the documentation, maybe "getting started with docker"?

Reply:
Now that I thought about it and looked at my local project, I remove the quotes from my env files...